lib/winappdriver.js

```diff
@@ -9,7 +9,7 @@ import B from 'bluebird';
 
 const REQD_PARAMS = ['app'];
 const DEFAULT_HOST = '127.0.0.1';
-const DEFAULT_PORT = "4823"; //  should be non-4723 to avoid conflict on the same box
+const DEFAULT_PORT = "4724"; //  should be non-4723 to avoid conflict on the same box
```

Comment:
hmm, just noticed we now have this in 2 places. What if we did:

```js
const DEFAULT_PORT = 4724;
```

here, and then in the referencing file:

```js
import { WinAppDriver, DEFAULT_PORT } from './winappdriver';
...
const WINAPPDRIVER_PORT = DEFAULT_PORT;
```

?